packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -958,6 +1008,7 @@ class _ViewContentState extends State<_ViewContent> {
                           child: SearchBar(
                             autoFocus: true,
                             constraints: headerConstraints ?? (widget.showFullScreenView ? BoxConstraints(minHeight: _SearchViewDefaultsM3.fullScreenBarHeight) : null),
+                            padding: widget.viewBarPadding ?? const MaterialStatePropertyAll<EdgeInsets>(EdgeInsets.symmetric(horizontal: 8.0)),
```

Comment:
Why does `viewBarPadding` needs to be a state property?

Reply:
I'm not sure. That's the current type of `SearchBar.padding`.

Reply:
Do we want to change it? It doesn't look like it needs to be a `WidgetStateProperty`.